src/components/GroupReport/OrgLegend.styles.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+export const OrgLegendContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+  margin-left: 10px;
+  margin-bottom: 5px;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const OrgLegendTitle = styled.span`
+  font-size: 1em;
+  font-weight: bold;
+  margin-right: 5px;
+`;
+
+export const OrgList = styled.div``;
+
+export const OrgItemContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  border: 2px solid black;
```

Comment:
Consider removing borders / additional geometry from elements which are not strictly necessary. Same for the OrgBadge